extensions/jms/src/main/java/org/mule/extensions/jms/api/operation/JmsConsume.java

```diff
@@ -79,8 +86,8 @@
                                                @Optional @Summary("The Type of the Consumer that should be used for the provided destination") ConsumerType consumerType,
                                                @Optional @Summary("The Session ACK mode to use when consuming a message") AckMode ackMode,
```

Comment:
I thought we were going to restrict the AckModes for Consume, since we are not going to intercept the end of the flow for auto ACK'in